src/libs/CMakeLists.txt

```diff
@@ -92,7 +92,9 @@ if(PortMidi_FOUND)
   target_link_libraries (midi ${PortMidi_LIBRARY})
 endif()
 
-find_package(SDL2)
+if (USE_SDL2)
+  find_package(SDL2)
+endif()
```

Comment:
Looking good, but one last nit 😉 - please wrap the whole section below in this if (USE_SDL2) clause.

Reply:
Okay, the change should be updated accordingly